src/components/SummaryContent.tsx

```diff
@@ -0,0 +1,70 @@
+import { Heading, Text, Box, Container, Stack, HStack, Flex } from "@chakra-ui/react";
+import { VFC } from "react";
+import { userPostTimes2022 } from "@/data/ranking";
+
+const SummaryContent: VFC = () => {
+  const UserPostTimes = userPostTimes2022;
+  return (
+    <Container py={2} bgColor="white" color="black">
+      <Stack my={2} margin={2}>
+        <Text fontSize="l">
+          レノファ山口の選手、サポーター、関係者の皆様、お疲れ様でした！
+          2022シーズンが終了しましたが、当サイトの運営にあたり、たくさんの方々にご協力いただいております。
+          特に、駐車場の混雑問題を解消するに向け、多くの投稿をしてくださった方々を以下にご記載いたします。
+          皆様、ご協力ありがとうございました。今後とも、引き続きよろしくお願いいたします。
+        </Text>
+      </Stack>
+      <Stack spacing={2} bgColor="orange.200" padding={3}>
```

Comment:
お礼文からのランキングが唐突なので、このようにランキングの旨の見出しを挟むとよいと思いました。
（Figmaにはなかったものですが）
```suggestion
      </Stack>
      <Text mt={8} mb={2} fontSize="md" textAlign="center">
        2022シーズンの投稿回数が多い方々
      </Text>
      <Stack spacing={2} bgColor="orange.200" padding={3}>
```

※この提案をそのままコミットとして取り込むことができます。